src/SyncMe/SyncMe.Common/Models/ConfigKey.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SyncMe.Models;
+
+public enum ConfigKey
+{
+    IsMute = 1,
+    IsDarkTheme = 2
```

Comment:
Do we still need this one?

Reply:
Let's create a new task to remove all `Dark Theme` related code including icons